SKILL.tsv

```diff
@@ -5734,302 +5734,302 @@ SKILL_20151102_005733	$[Merkabah] petrifies [Cursed] enemies for 3 seconds.
+SKILL_20151224_005796	Gem penalty stage reduction: #{CaptionRatio}#
+SKILL_20151224_005797	{#993399}{ol}[Magical]{/}{/}{nl}Stab the enemy with a bone knife. If the enemy is a Beast, Demon or Insect-type, blood will spew out which will heal your allies.
+SKILL_20151224_005798	Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio}#%{nl}{#339999}{ol}[Bleed]{/}{/} Duration: 5 second{nl}HP Recovery: #{CaptionRatio2}#
+SKILL_20151224_005799	{#993399}{ol}[Magical]{/}{/}{nl}Attack and drain the enemy's life force, while recovering health at the same time.
+SKILL_20151224_005800	Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio}#%{nl}Suction: #{CaptionRatio2}#%{nl}SP consumption per 0.4 second: #{CaptionTime}#
+SKILL_20151224_005801	{#993399}{ol}[Magical]{/}{/}{nl}Summon a carved, pointed bone called a 'Kundela'. The Kundela points towards enemies in range, attacking them with bone shards and cursing them.
+SKILL_20151224_005802	Attribute Attack: #{CaptionRatio}#%{nl}Bone Pointing duration: #{CaptionRatio2}#second{nl}{#339999}{ol}[Curse]{/}{/} Duration: 7 second
+SKILL_20151224_005803	{#993399}{ol}[Magical]{/}{/}{nl}Cut your enemy using a cursed bone. Whilst the bone is stuck in the enemy, they will continually decay.
+SKILL_20151224_005804	Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio}#%{nl}Duration:#{CaptionRatio2}#second
+SKILL_20151224_005805	{#993399}{ol}[Magical]{/}{/}{nl}Put on shoes made of mixed hair and blood. Walk while leaving bloody footprints in the ground, which curses any enemies who come into contact with them.
+SKILL_20151224_005806	Maximum Kurdaitcha duration: #{CaptionRatio}#second{nl}Moving speed reduction: 15{nl}{#339999}{ol}[Curse]{/}{/} Duration: 7 second
+SKILL_20151224_005807	{#993399}{ol}[Magical] - [Dark]{/}{/}{nl}Summon a pillar of darkness, which causes enemies in contact to suffer continuously.
+SKILL_20151224_005808	Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio}#%{nl}Duration: #{CaptionTime}#second{nl}#{CaptionRatio2}#Additional damage per second: 
+SKILL_20151224_005809	{#993399}{ol}[Magical] - [Dark]{/}{/}{nl}Summon evil spirits from the death of enemies. Summoned spirits remain in the field damage enemies in contact with them.
+SKILL_20151224_005810	{#993399}{ol}[Magical] - [Dark]{/}{/}{nl}Summon spirits to protect the caster by surrounding them.
+SKILL_20151224_005811	Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio}#%{nl}Duration: 30 second
+SKILL_20151224_005812	{#993399}{ol}[Magical] - [Dark]{/}{/}{nl}Summon demons of hatred to attack enemies. Prioritize the enemy hit by this magic and hit with additional damage by holy attacks.
+SKILL_20151224_005813	Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio}#%{nl}Stigma duration: 5 second
+SKILL_20151224_005814	Evil Sacrifice
+SKILL_20151224_005815	{#993399}{ol}[Magical]{/}{/}Direct spirits to a location, and dedicate enemies in range as a sacrifice. Summoned spirits will move towards that area.
```

Comment:
[Magical] -> [Magic]

Direct haunting spirits to a targeted area to dedicate enemies within range as a sacrifice.